raw_topic: "/raw_frame"

port_name: "/dev/ydlidar"
baudrate: 128000

frequency_read: 5000

scan_topic: "/lidar/scan_frame" 

max_angle: 225
min_angle: 135

max_distance: 3
min_distance: 0.13

frame_id: "laser_frame"
